bebop_driver/src/bebop_driver_nodelet.cpp

```diff
@@ -380,6 +393,19 @@ void BebopDriverNodelet::TakeSnapshotCallback(const std_msgs::EmptyConstPtr &emp
   }
 }
 
+void BebopDriverNodelet::SetExposureCallback(const std_msgs::Float32ConstPtr& exposure_ptr)
+{
+  try
+  {
+    ROS_INFO("Setting exposure to %f", exposure_ptr->data);
```

Comment:
It would be great to add a `CLAMP` hear not to let any number outside the [-3.0..3.0] range to be passed.